### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by kreezxil** [BloodWorkXGaming/ExNihiloCreatio#137]
How do make the mod use new registries?

For instance I want to lower the chance that coal comes out of a mesh.

I borrowed the gravel definition from stone block and adjusted the coal return values, but no matter what for the flint mesh it still shows a 12% chance and I still get 8 coal average for a full stack of gravel.

My json file.
http://paste.ubuntu.com/p/VC9XzM9Trq/

**Comment by SirLyle** [comment]
Did you enable json configs in the main config file?

**Comment by kreezxil** [comment]
Thanks for the response, I did not enable that setting because I did not easily find that setting. With some help from the MMD group on Discord I was able to find the setting buried 75% down into the config file under the misc {} section.

Speaking of which, considering that setting is extremely important, it would be nice to have it an similar items found near the top of the config file vs way down where it is.

When you have 100's of configs to look at things get quite blurry.

Or better yet, a page on the curseforge page that tells us where and how to load the customizations.
